libunwind/src/config.h

```diff
@@ -223,6 +223,21 @@
     } while (0)
 #endif
 
+#define _LIBUNWIND_TRACING_DWARF_EVAL (0)
```

Comment:
Maybe it would be best to follow the same pattern as above and allow turning it on for debug builds? But seeing as it's always been hardcoded to false, maybe the current preprocessor define makes more sense. It should be very rare that this kind of tracing is needed and a debugger might be better.

Reply:
Yeah, I figured if anyone ever actually needed to turn this on for debug builds in general, they'd have added that option. It seemed to me like this is more just some convenience thing for whoever was working on that code. I didn't want to couple this to `NDEBUG` or some external function like the other stuff does since this output so far wasn't turned on that way either. And adding a whole CMake build option just for this seemed like overkill. Given the way it's been used so far, doing it via a CMake option would likely get in the way more than anything else since that would force a wider rebuild every time you switch. So I just kept the overall behavior as-is. With the macro in the config header now, it will be trivial to add a build flag or something, should the need ever arise.

Reply:
In fact, since this is more some kind of internal feature, maybe it shouldn't be in the config header at all but rather just defined and undef'ed again inside the original function? Though having all the stuff to do with logging defined in a similar fashion in one place is probably preferable…